agent/main.go

```diff
@@ -102,7 +121,12 @@ func main() {
 	_ = corev1.AddToScheme(scheme)
 	_ = clusterv1.AddToScheme(scheme)
 
-	logger := klogr.New()
+	// setting info log verbosity level at 1
+	logger := klogr.New().V(1)
```

Comment:
shouldn't this be called post the flag set?

Reply:
I think the setting up of the logger verbosity is independent of the log visibility level. That is, irrespective of what the "`v`" flag is set at, the logger will log information at `verbosity level 1`

Reply:
i thought "v" flag is for verbosity level https://github.com/kubernetes/klog/blob/main/klog.go#L464

Reply:
So there are two parameters, `logger := klogr.New().V(1)` defines the level at which our info logs are generated while the `v` flag determines up to which level should the logs should be printed.

Eg, say we have `logger.V(1).log("blah 1")`, `logger.V(2).log("blah 2")` and `v` is set at 2, then only `"blah 2"` will be printed

In our case, in the above code block, we are setting the default verbosity level at which all logs are generated to 1.